commands.js

```diff
@@ -95,10 +95,10 @@ var commands = exports.commands = {
 	requesthelp: 'report',
 	report: function (target, room, user) {
 		if (room.id === 'help') {
-			this.sendReply("Ask one of the Mods in the Help room.");
+			this.sendReply("Ask one of the Moderators (@) in the Help room.");
 		} else {
-			this.sendReply("Ask a Mod in the Help room.");
 			this.parse('/join help');
+			user.popup("Ask one of the Moderators (@) in the Help room.");
```

Comment:
But if a message would be closed, and user is already in Help, nothing would happen, and this could actually confuse an user.